### PR TITLE
Story/timur_abd

### PR DESCRIPTION
**Pull request by Akamemz**
Please discard my previous branch as i have some issue with "Story/timur"

### CODE REVIEW

**Review by GeoffreyXue — CHANGES_REQUESTED** [review]
Everything looks good, just 2 changes:
- 1 minor change to the Variable Used header
- Could all of the content be moved into the Stories folder within affordable and clean energy?

File: content/topics/affordable-and-clean-energy/story_Timur.md
[the remark this answers is not in the document]
Comment (PranavC10) [reply content/topics/affordable-and-clean-energy/story_Timur.md:1]: @Akamemz Hi Timur, Please make these changes.

[the remark this answers is not in the document]
Comment (Akamemz) [reply content/topics/affordable-and-clean-energy/story_Timur.md:1]: Hello @GeoffreyXue @PranavC10, thanks for a feedback, i have adjusted necessary changes that were requested.

**Review by GeoffreyXue — APPROVED** [review]
Looks good! I notcied you moved the index.md file into the folder too - I've just gone ahead reverted that change for you.

**Pull request merged**